src/librustc_trans_utils/trans_crate.rs

```diff
@@ -226,7 +226,7 @@ impl TransCrate for MetadataOnlyTransCrate {
         };
     }
     fn provide_extern(&self, providers: &mut Providers) {
-        self.provide_local(providers)
+        self.provide(providers)
```

Comment:
You shouldn't need to do anything here, this is redundant.

Reply:
You mean i should remove this line?

Reply:
Yes.